peps/pep-0724.rst

```diff
@@ -0,0 +1,305 @@
+PEP: 724
+Title: Stricter Type Guards
+Author: Rich Chiodo <rchiodo at microsoft.com>,
+        Eric Traut <erictr at microsoft.com>,
+        Erik De Bonte <erikd at microsoft.com>,
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: https://mail.python.org/archives/list/typing-sig@python.org/thread/EMUD2D424OI53DCWQ4H5L6SJD2IXBHUL/
```

Comment:
This should be the first Post-History entry; Discussions-To should be a new thread

Reply:
Oh okay. Will change. How do I create the new thread? Is there a standard format for such a thing?